tools/codesigningtool/codesigningtool.py

```diff
@@ -120,16 +123,100 @@ def _certificate_fingerprint(identity):
   fingerprint = fingerprint.replace(":", "")
   return fingerprint
 
+def _certificate_common_name(cert):
+  _, subject, _ = execute.execute_and_filter_output([
+    "openssl",
```

Comment:
It might be worth pointing explicitly to `/usr/bin/openssl` here, in the off chance that someone consuming these rules happens to have a custom `openssl` in their PATH.

Reply:
This is done